Mathlib/Probability/Moments.lean

```diff
@@ -180,13 +180,26 @@ theorem mgf_pos [IsProbabilityMeasure μ] (h_int_X : Integrable (fun ω => exp (
     0 < mgf X μ t :=
   mgf_pos' (IsProbabilityMeasure.ne_zero μ) h_int_X
 
+lemma mgf_id_map {Ω : Type*} {m : MeasurableSpace Ω} {μ : Measure Ω}
+  (X : Ω → ℝ) (hX : AEMeasurable X μ) : mgf id (μ.map X) = mgf X μ := by
```

Comment:
All those arguments are already declared in a `variable` line above. Also `X` should be implicit because it can be inferred from `hX`.
```suggestion
lemma mgf_id_map (hX : AEMeasurable X μ) : mgf id (μ.map X) = mgf X μ := by
```